Dockerfile

```
@@ -1,8 +1,7 @@
 # syntax = docker/dockerfile:1
 
 # Adjust NODE_VERSION as desired
-ARG NODE_VERSION=22.13.1
-FROM node:${NODE_VERSION}-slim AS base
+FROM node:22.13.1-slim
```

```suggestion
FROM node:22.13.1-slim AS base
```

We're using multi-stage docker build in this Dockerfile, and removing the `AS <name>` causes build to break.